templates/layouts/Default:-petition-and-scroll-to-share-greenpeace-refactor.liquid

```diff
@@ -25,7 +25,7 @@
 
   <div class="center-content__fixed-right center-content--push-down" data-transition-id="petition-wrapper">
     {% include 'Petition Sidebar Refactor',
-      transition: 'petition-wrapper:share-prompt',
+      transition: 'petition-wrapper:share-prompt__petition-mobile-wrapper:share-prompt',
```

Comment:
I'd rather not put show/hide logic in the same place a scroll to, since declaring transitions will become complicated or broken. Let's just set up an event listener since we have a `complete` event for the petition plugin (using the plugin api). Ping me and we go through this.

Reply:
Changed the logic to be using events emit instead of transition flow